framework/wazuh/core/utils.py

```diff
@@ -145,21 +145,31 @@ def process_array(array: list, search_text: str = None, complementary_search: bo
         List of fields allowed to select from.
     filters : dict
         Defines required field filters. Format: {"field1":"value1", "field2":["value2","value3"]}
+    distinct : bool
+        Look for distinct values.
 
     Returns
     -------
     dict
         Dictionary: {'items': Processed array, 'totalItems': Number of items, before applying offset and limit)}
     """
     if not array:
-        return {'items': list(), 'totalItems': 0}
+        return {'items': [], 'totalItems': 0}
```

Comment:
Performance-wise it's better to use empty lists `[]` instead of `list()` (https://stackoverflow.com/a/5790954)